src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -1196,10 +1196,10 @@ def test_webapp_ssl(self, resource_group, resource_group_location):
 
 class WebappSSLImportCertTest(ScenarioTest):
     @ResourceGroupPreparer(location=WINDOWS_ASP_LOCATION_WEBAPP)
-    def test_webapp_ssl_import(self, resource_group):
+    @KeyVaultPreparer(location=WINDOWS_ASP_LOCATION_WEBAPP, name_prefix='kv-ssl-test', name_len=20)
+    def test_webapp_ssl_import(self, resource_group, key_vault):
```

Comment:
`key_vault` is usually used as a single name.

Reply:
I just keep the origin definition within `KeyVaultPreparer` to make this PR change as small as possible.